### PR TITLE
feat: improve fs example

### PR DESCRIPTION
**Pull request by sujit-baniya**
Optimizes example for Static and StaticFS using html and assets as example and fixes https://github.com/cloudwego/hertz/issues/400

### CODE REVIEW

File: file/html-fs/main.go [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package main
+
+import (
```
Comment (justlorain) [inline file/html-fs/main.go:19]: Use the following import style, other ones should be the same:
```go
import (
        "context"
        "strings"

        "github.com/cloudwego/hertz/pkg/app"
	"github.com/cloudwego/hertz/pkg/app/server"
)
```

**Comment by justlorain** [comment]
ci need to resolve

**Comment by sujit-baniya** [comment]
@li-jin-gou Got error on CI. Not sure if I understand the error correctly. Can you please check and suggest

**Comment by li-jin-gou** [comment]
> @li-jin-gou Got error on CI. Not sure if I understand the error correctly. Can you please check and suggest

OK

File: file/html-fs/main.go [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+		prefix = "/"
+	}
+	// Prefix always start with a '/' or '*'
+	if prefix[0] != '/' {
+		prefix = "/" + prefix
+	}
+
+	// Strip trailing slashes from the root path
+	if len(root) > 0 && root[len(root)-1] == '/' {
+		root = root[:len(root)-1]
```
Comment (li-jin-gou) [inline file/html-fs/main.go:58]: this value of root is never used！

**Comment by sujit-baniya** [comment]
@li-jin-gou removed unused variable. But can you suggest how you figured it out from the error?

**Comment by li-jin-gou** [comment]
refer to https://github.com/cloudwego/hertz-examples/actions/runs/3510397521/jobs/5882283983

image

**Review by welkeyever — APPROVED** [review]
LGTM, thx~

**Pull request merged**